client/components/payment-method-logos/style.scss

```diff
@@ -0,0 +1,21 @@
+.connect-account-page__payment-methods--logos {
+	display: flex;
+	align-items: center;
+	border: 1px solid $gray-300;
+	border-bottom: none;
+	padding: 12px;
+	img {
+		margin-right: 8px;
```

Comment:
I have observed that the borders of the logos included in the design are missing:

<img width="1200" alt="main-component-1" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/46ff355e-a260-4027-a238-84cb2b256742">

```suggestion
		margin-right: 8px;
		border: 1px solid $gray-300;
```

Reply:
changed in [a8827d3](https://github.com/Automattic/woocommerce-payments/pull/8977/commits/a8827d358ba715c9bb02527de2e03d93e75ad6f7) . thx.